src/plugins/intel_cpu/src/dnnl_extension_utils.cpp

```diff
@@ -98,7 +99,7 @@ InferenceEngine::Precision DnnlExtensionUtils::DataTypeToIEPrecision(memory::dat
         case memory::data_type::u4:
             return InferenceEngine::Precision::U4;
         case memory::data_type::undef:
-            return InferenceEngine::Precision::UNSPECIFIED;
+            return InferenceEngine::Precision::UNSPECIFIED; // TODO: How about reverse conversion of Precision::STRING?
```

Comment:
Probably part of the CPU hack -- don't know how are you going to represent string tensor inside.